pgdrive/envs/pgdrive_env.py

```diff
@@ -43,7 +43,7 @@ def default_config() -> PgConfig:
 
             # ===== Traffic =====
             traffic_density=0.1,
-            traffic_mode=TrafficMode.Add_once,
+            traffic_mode=TrafficMode.Trigger,
```

Comment:
It's really necessary to change the default config?

Reply:
I will reset it